elytron/src/main/java/org/wildfly/extension/elytron/_private/ElytronSubsystemMessages.java

```diff
@@ -346,6 +346,9 @@ public interface ElytronSubsystemMessages extends BasicLogger {
     @Message(id = 47, value = "LDAP Realm is configured to use direct-verification and user-password-mapper which is invalid configuration.")
     void ldapRealmDirectVerificationAndUserPasswordMapper();
 
+    @Message(id = 48, value = "String representation of an X.500 distinguished name is required. String must be in following format: 'CN=cName, OU=orgUnit, O=org, L=city, S=state, C=countryCode'.")
```

Comment:
I don't think we need to include an example of the format here. Instead, I think it would be good to include information from the actual exception message (`e.getMessage()`), e.g.,

A string representation of an X.500 distinguished name is required: %s

Reply:
Thanks @fjuma @yersan, updated.